game/scorewindow.hh

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+class Database;
+
+#include "animvalue.hh"
+#include "instruments.hh"
+#include "progressbar.hh"
+#include "texture.hh"
+#include "theme.hh"
+
+/// shows score at end of song
+class ScoreWindow {
+  public:
+	ScoreWindow(Instruments& instruments, Database& database);
+    
+	/// draws ScoreWindow
+	void draw();
+	bool empty();
+    
+  private:
+	Database& m_database;
```

Comment:
should/can this be const?

Reply:
Should, yes, but can not, because ScoreWindow sort the scores inside the Database class. Bad designed but I want not to refactor it at this point. Except @OznOg you has an idea for an easy solution.